### ISSUE TITLE
Frequent BSODs on latest drivers

### ISSUE THREAD

**Issue by kasperl** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#874]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

Grand Theft Auto V

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

Intel i7-13700F

### Graphic Card [Required]

Intel Arc A750 LE

### GPU Driver Version [Required]

32.0.101.6314

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build [Required]

Windows 11 24H2

### Intel System Support Utility report

[intel-ssu-051224.txt](https://github.com/user-attachments/files/18021163/intel-ssu-051224.txt)

### Description and steps to reproduce [Required]

I do not have any specific steps to reproduce the issue, but it typically happens while playing Grand Theft Auto V.

I have tried multiple different driver versions, but the last good version for me is 6078 WHQL and I haven't seen any BSODs on that version. I've tried 6127, 6129, 6297, and now 6314 and experienced multiple BSODs on all of them. I have gone back to 6078.

The issue does not seem specific to GTA and it has been reported by others. Here is a list of relevant discussions on Reddit:

https://www.reddit.com/r/IntelArc/comments/1h6s08b/average_crashes_within_a_week/
https://www.reddit.com/r/IntelArc/comments/1gx9w5b/bsods_on_newer_drivers/
https://www.reddit.com/r/IntelArc/comments/1gze5ir/my_pc_is_randmoly_crashing_with_a_bsod_saying/
https://www.reddit.com/r/IntelArc/comments/1gvbfn9/intel_arc_graphics_driver_3201016299/

### Game graphic quality [Required]

- [ ] Low
- [X] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

Off

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

FXAA: Off
MSAA: X2
Texture Quality: Normal
Shader Quality: Normal
Shadow Quality: Normal
Reflection Quality: Normal
Reflection MSAA: Off
Water Quality: Normal
Particles Quality: Normal
Grass Quality: Normal
Soft Shadows: Sharp
Post FX: Normal
Anisotropic Filtering: Off
Ambient Occlusion: Off
Tessellation: Off

### Crash dumps [Required, if applicable]

Windows minidump:

[120524-7078-01.dmp](https://github.com/user-attachments/files/18021168/120524-7078-01.dmp)

**Comment by Felipe-Intel** [comment]
Hello @kasperl 
How long it takes for your PC to crash/BSOD after you start playing? Is it consistent (e.g. always around 30 min) or is it irregular (from 10 min to 1 hr+)?
Could you please refer to [these instructions](https://community.intel.com/t5/Intel-ARC-Graphics/Need-help-Reporting-a-bug-or-issue-with-Arc-GPU-PLEASE-READ-THIS/td-p/1494429#:~:text=Crashing/freezing%20issues/BSOD%3A) and check if the crash produces other DMP files? If you decide to go ahead, remember to revert the changes after testing.

**Comment by kasperl** [comment]
Hi @Felipe-Intel! 

The time it takes for the PC to hit the BSOD varies. Today, after upgrading to 6314 WHQL last night, it took around an hour - maybe a little bit less. Sometimes, it takes a few hours. Over a week on the affected drivers, I typically see ~2 crashes per day and Grand Theft Auto V seems to be the "easiest" way of reproducing it.

Using the 6078 WHQL driver, I have seen no issues over several weeks, so it definitely smells like a driver regression.

I'll look at the instructions you sent. I hope the minidump I already provided is at least somewhat useful to guide the engineers in the right direction.

Thanks!

**Comment by kasperl** [comment]
Pretty sure GTA V uses Direct X 11, but I can double check in the settings later. I forgot to add that information earlier.

**Comment by Denizeri24** [comment]
I get black screen randomly too. I disabled ALL OCs like RAM / CPU and still get crash (gpu is not oc'd). Crash time is changing, sometimes 5 - 6 hour, sometimes 2 min after pc reboot. (playing while LoL)

**Comment by rmorera-intel** [comment]
Thanks for the mini-dump, our debug team is looking into this already (for reference it is under internal bug ID 18040523387).

In parallel I'm also trying to reproduce the issue (I'm planning to invest some serious hours into the new Indiana Jones game and will also hit GTA 5).

**Comment by kasperl** [comment]
@rmorera-intel Sounds great! I'm a low-level software engineer myself, so let me know if I can help in any way. If the full dumps turn out to be necessary, I'll try to provide them quickly - even though my son will hate having to endure more crashes for a day or two :)

**Comment by rmorera-intel** [comment]
Thanks I'll keep that offer in mind. For now please keep using 6078 (if you don't mind) so we can get a solid foundation on whether or not the system works fine with it (i.e. there is always the possibility you may have a faulty GPU so if there is a BSOD or crash with 6078 then we'll pretty much confirm it)

**Comment by kasperl** [comment]
@rmorera-intel Got it. I've been running 6078 for some weeks now and I haven't seen a single BSOD in the logs.

**Comment by kasperl** [comment]
@rmorera-intel Do you recommend that I upgrade to the latest WHQL driver (32.0.101.6325) and give that a try - or would you rather have me stick to 6078 for a little while longer?

Looks like there are lots of moving parts to the new driver. Fantastic to follow the progress :)

**Comment by davidrobertbrt** [comment]
Bumping this issue thread because I have the same issue with the latest driver (as of the writing of this comment the driver version in question is: 32.0.101.6325 WHQL).
I downgraded to the same driver mentioned by @kasperl (6078) and the crashes went away. I am attaching the dump file for further investigation. 
[121524-9937-01.dmp](https://github.com/user-attachments/files/18155077/121524-9937-01.dmp)

_One issue I would like to point out with the latest driver is that sometimes changing from one displayport to another on my GPU causes inconsistent brightness (as in one port is outputing more brightness then the other which doesn't happen on 6078)_

**Now, while the BSODs are gone on this version, I would like to point out one issue that still remains on both driver versions:**
Counter Strike 2 suddently freezes for 5 seconds and then the game returns to normal. This happens only once during gameplay and it is inconsistent. The longest I have gone without it crashing was 90 minutes at best. _Attached is a zip with the minidump provided by the game._ [frequent_bsods_cs2_github_issue874.zip](https://github.com/user-attachments/files/18155204/frequent_bsods_cs2_github_issue874.zip)

I have tested the GPU on two separate systems and all the issues mentioned above are still present.

For reference I am posting my PC specs to end this comment
```
CPU: Intel i7 14700K
GPU: Asrock Challenger D A750 8GB
RAM: 32GB Corsair CMK32GX5M2B5200Z40
PSU: bequiet! straight power 12 850W
```

**Comment by Pashamolochko** [comment]
I also have blue screens on the latest intel driver,the time varies,the system can work stably for 7 hours and fail, or 10-20 minutes and also fail,it is not clear what the reason is,the error SYSTEM_THREAD_EXCEPTION_NOT_HANDLED (7e)
![gt89dy-3ubo](https://github.com/user-attachments/assets/3e814a96-8b9d-4d40-86b5-8854592371ca)
![vV57YybxXNc](https://github.com/user-attachments/assets/44d98c71-a649-49d6-956c-dba641dd47c8)

**Comment by Pashamolochko** [comment]
[121724-6671-01.zip](https://github.com/user-attachments/files/18186074/121724-6671-01.zip)

**Comment by Pashamolochko** [comment]
[121724-6687-01.zip](https://github.com/user-attachments/files/18186155/121724-6687-01.zip)

**Comment by Pashamolochko** [comment]
[121824-6968-01.zip](https://github.com/user-attachments/files/18186160/121824-6968-01.zip)

**Comment by Pashamolochko** [comment]
I bought a new GPU a couple of days ago,this is Intel Arc A580 ASRock Challenger,before that there was a770 until September of this year,there were no similar problems on it for 2 years,other system characteristics:CPU:AMD Ryzen 7 5700X3D,Motherboard:ASUS ROG STRIX B550-A GAMING

**Comment by rmorera-intel** [comment]
Hi everyone I got an update.
Debug has been trying to reproduce the BSODs to no avail and without a successful way to reproduce the BSOD I was told we won't be able to proceed further.

Try updating the driver to our latest [32.0.101.6332](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) and if you continue to see BSODs then it may be time to do warranty on the GPU - I know sounds odd since the same hardware seems to work fine on older driver but the failure/issue repro rate is low so hardware is not entirely ruled-out.

**Comment by kasperl** [comment]
Hey @rmorera-intel! Thanks for the update.

I'll use DDU v18.0.8.9 as part of the upgrade from 32.0.101.6078 to 32.0.101.6332. I'll let you know how that goes and proceed with the warranty option if the BSODs persist.

**Comment by Denizeri24** [comment]
> Hi everyone I got an update. Debug has been trying to reproduce the BSODs to no avail and without a successful way to reproduce the BSOD I was told we won't be able to proceed further.
> 
> Try updating the driver to our latest [32.0.101.6332](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) and if you continue to see BSODs then it may be time to do warranty on the GPU - I know sounds odd since the same hardware seems to work fine on older driver but the failure/issue repro rate is low so hardware is not entirely ruled-out.

Issue still persist. Thats %100 driver based problem, I don't get any errors with older drivers. If you guys can't fix that problem, just give me my money back. I bought ~2 years ago A770.

**Comment by rmorera-intel** [comment]
@Denizeri24 if your Arc GPU is the Intel A770 Limited Edition you can [contact Intel Customer Support ](https://www.intel.com/content/www/us/en/support/contact-us.html)to pursue warranty/refund.
I still suggest checking for a warranty exchange since this could still be hardware - if your GPU is from one of our partners you can contact them directly to explore warranty options.

**Comment by Denizeri24** [comment]
I don't believe thats a hardware issue, I don't get crash / freeze when using old drivers (I'm using this card like ~2 year), also I never OC'd the card..

(770 LE 16GB)

**Comment by rmorera-intel** [comment]
Do you have any way to reproduce the BSOD so we can replicate it here? is it crashing only at random when you play League of Legends (LOL) or any other game or app also BSODs?

**Comment by Denizeri24** [comment]
> Do you have any way to reproduce the BSOD so we can replicate it here? is it crashing only at random when you play League of Legends (LOL) or any other game or app also BSODs?

I'm only play LoL and crashing randomly when playing. Also limiting power to 95W fixing this issue (I BELIVE).

**Comment by Pashamolochko** [comment]
> > Есть ли у вас способ воспроизвести BSOD, чтобы мы могли повторить его здесь? Он вылетает только случайным образом, когда вы играете в League of Legends (LOL) или любую другую игру или приложение, также выдающее BSOD?
> 
> Я играю только в LoL и вылетает случайным образом во время игры. Также ограничение мощности до 95 Вт исправит эту проблему (Я ВЕРЮ).

hi, try to test a clean installation without intel arc control and the new intel software, and also remove intel driver and support assistant, install just the driver itself, and try to test this method.

**Comment by kasperl** [comment]
@rmorera-intel Just had the first BSOD on the latest driver (6332) when starting up Red Dead Redemption 2. 

I can forward the dump, but I guess this means that I should start the RMA process? What is the best way to convince the retailer where the card was bought that we've already investigated the issue and that you (Intel) have concluded that it should be treated as malfunctioning hardware?

The card was bought as part of a pre-assembled system, but if it is more convenient to go through Intel's online warranty system, I can try that as well. Please advice.

**Comment by rmorera-intel** [comment]
@kasperl go ahead and check with your place of purchase first - you can refer them to this report. If they pushback simply contact [Intel Customer Support](https://www.intel.com/content/www/us/en/support/contact-us.html) directly.

**Comment by kasperl** [comment]
@rmorera-intel I'll give it a go with the place of purchase, but I already got some light push back and they don't seem to value the report here much :)

On a related matter, I have tried decoding the Serial Number and Product ID from the QR code on the GPU to check on warranty coverage. I've scanned the code and it gives me a string like ADNFxxxxSKUyyyyyyyyyy (4 times x, 10 times y). First of all, does that look like a correct Serial Number? I assume this is the full Serial Number.

The support article is quite unhelpful in how to find the Product ID from that:
https://www.intel.com/content/www/us/en/support/articles/000090979/graphics.html

Maybe someone here can explain how to proceed? 🤞

**Comment by rmorera-intel** [comment]
@kasperl you got it, the ADNFxxxxSKUyyyyyyyyyy is the GPU's serial number.
The Product ID is located next to the QR code on the bracket of the GPU (next to the video outputs).

**Comment by Denizeri24** [comment]
Latest driver, not fixed.

Anyway I created a refund ticket, I will go with RTX 5070..

**Comment by rmorera-intel** [comment]
Hey everyone I'm re-opening this bug report. Upon further inspection this seems to be a known issue impacting a small subset of users and our dev team is going to resume work on this.

As soon as there is an update I'll communicate it here.

**Comment by rmorera-intel** [comment]
Refer to official article: https://www.intel.com/content/www/us/en/support/articles/000100502/graphics.html

I'm going to close this as a duplicate of https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/issues/967

As soon as there is an update on this issue we will update the aforementioned article and #967
